### PR TITLE
PTU in case of failed retry strategy during previour IGN_ON (SDL.PolicyUpdate)

### PR DESCRIPTION
**Pull request by istoimenova**
003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua - Script is renamed to 
003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle_ExternalProprieatry

To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+local mobile_session = require('mobile_session')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+testCasesForPolicyTable:flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY()
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua]: @istoimenova Call of function should be updated to be included in Test.

File: test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  local correlationId = self.mobileSession1:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application2.appName } })
+  :Do(function(_,data)
+      hmi_app2_id = data.params.application.appID
+      self.applications[config.application2.registerAppInterfaceParams.appName] = data.params.application.appID
+
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+
+      testCasesForPolicyTableSnapshot:create_PTS(true, {
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua]: @istoimenova create_PTS should be renamed as verify_PTS

File: test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle()
+  -- hmi_app1_id = self.applications[config.application1.registerAppInterfaceParams.appName]
+  local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.appName } })
+  :Do(function(_,data)
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+
+      testCasesForPolicyTableSnapshot:create_PTS(true, {
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua]: @istoimenova create_PTS should be renamed as verify_PTS

File: test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua [hunk 4]
```diff
@@ -0,0 +1,182 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] Policy Table Update in case of failed retry strategy during previour IGN_ON (SDL.PolicyUpdate)
+-- [HMI API] PolicyUpdate request/response
+--
+-- Description:
+-- SDL should request in case of failed retry strategy during previour IGN_ON
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua]: Please add what is requested

File: test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+-- 1. Used preconditions
+-- SDL is built with "-DEXTENDED_POLICY: EXTERNAL_PROPRIETARY" flag
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua]: Please check if flag is -DEXTENDED or just EXTENDED

Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua]: @maria-mih, according to specification is DEXTENDED, in real usage of Cmake is EXTENDED

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**